ansible/hosts.yml

```diff
@@ -0,0 +1,12 @@
+all:
+  children:
```

Comment:
I'd add some comments here and/or elsewhere that just explain what JMeter means by "client" and "server", since it's always a bit surprising -- to me, at least.

Reply:
Done.